### PR TITLE
Untested code tidying.

### PR DESCRIPTION
**Pull request by ewels**
Main things:

* Stick to lower case variable names with only letters and underscores (not hyphens)
* Making commands multi-line helps with spotting typos ;)
* I prefer not to escape variables with `{}` when they're simple
  * eg. `$var`
  * I usually do escape `${var.something}` or `${var},` (eg. when no whitespace separation)
* Outputs should always go into channels, even if those channels aren't used (I think)

### CODE REVIEW

File: conf/uppmax.config [hunk 1]
```diff
@@ -19,22 +19,22 @@ process {
...
 
-    params {
-        references{
-            fusioncatcer_data = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/ensembl_v83/'
-            star_fusion_reference  = '/proj/b2013064/private/nobackup/rickard/STAR_fusion_refrence/GRCh37_gencode_v19_CTAT_lib_July272016_prebuilt'
-            fusioncatcher_data_dir = '/sw/apps/bioinfo/FusionCatcher/0.99.5a/milou/data/current'
-            }
-    }
+params {
+  references{
```
Comment (ewels) [inline conf/uppmax.config:35]: These don't seem to be used anywhere?

File: main.nf [hunk 2]
```diff
@@ -21,78 +21,66 @@ version = '0.1'
...
-params.reverse_stranded = false
-params.unstranded = false
-params.star_index = params.genome ? params.genomes[ params.genome ].star ?: false : false
-params.fasta = params.genome ? params.genomes[ params.genome ].fasta ?: false : false
-params.gtf = params.genome ? params.genomes[ params.genome ].gtf ?: false : false
-params.bed12 = params.genome ? params.genomes[ params.genome ].bed12 ?: false : false
-params.hisat2_index = params.genome ? params.genomes[ params.genome ].hisat2 ?: false : false
-params.multiqc_config = "$baseDir/conf/multiqc_config.yaml"
-params.splicesites = false
-params.download_hisat2index = false
```
Comment (ewels) [inline main.nf:35]: I prefer to delete stuff like this unless we're sure that it's going to be used soon. Can always add it back easily enough.

File: main.nf [hunk 3]
```diff
@@ -21,78 +21,66 @@ version = '0.1'
...
-        
     input:
-    set val (name), file(read1), file(read2) from read_files_STAR-fusion   
+    set val (name), file(read1), file(read2) from read_files_star_fusion
 
     output:
-    '*final.abridged*'
-    'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
-    
-    when: STAR == true
```
Comment (ewels) [inline main.nf:67]: You were missing `params.` so I'm pretty sure this was always undefined..?

File: main.nf [hunk 4]
```diff
@@ -21,78 +21,66 @@ version = '0.1'
...
-    when: STAR == true 
-    
+    file '*final.abridged*' into star_fusion_abridged
+    file 'star-fusion.fusion_candidates.final.abridged.FFPM' into fusion_candidates
+
+    when: params.star
+
+    script:
     """
-    STAR-Fusion --genome_lib_dir ${star_fusion_refrence} -_left_fq ${read1} --right_fq ${read2}  --output_dir ${star-fusion_outdir}
```
Comment (ewels) [inline main.nf:70]: Note typo `-_left_fq`

File: main.nf [hunk 5]
```diff
@@ -21,78 +21,66 @@ version = '0.1'
...
 
     input:
-    set val (name), file(read1), file(read2) from fusionInspector-reads 
-    file fusion_candidates 
-    
+    set val (name), file(read1), file(read2) from fusion_inspector_reads
+    file fusion_candidates
+
     output:
+    file '*' into fusioninspector_results
```
Comment (ewels) [inline main.nf:70]: You had `output:` here with nothing in the section, which was probably confusing the hell out of Nextflow..

File: main.nf [hunk 6]
```diff
@@ -101,33 +89,27 @@ process FusionInspector{
...
     fusioncatcher \
-    -d ${fusioncatcher_data_dir} \
-    -i  ${read1},${read2} \
-    --threads ${task.cpus} \
-    --${SENSITIVITY} \
-    -o ${SAMPLE_NAME}/
+        -d $fusioncatcher_data_dir \
+        -i ${read1},${read2} \
+        --threads ${task.cpus} \
+        --$SENSITIVITY \
```
Comment (ewels) [inline main.nf:109]: Quite a lot of these variables don't exist, but I guess you know that already..

File: conf/uppmax.config (hunk 1 above)
Comment (Hammarn) [reply conf/uppmax.config:35]: Note yet, or the `fusioncatcher_data_dir` is supposed to be used in the `fusioncatcher` process. Maybe I mad a typo? I tend to do that

File: main.nf (hunk 2 above)
Comment (Hammarn) [reply main.nf:35]: Fair point

File: main.nf (hunk 3 above)
Comment (Hammarn) [reply main.nf:67]: I don't think so, I was looking at CAW when I wrote this: https://github.com/SciLifeLab/CAW/blob/master/main.nf#L244
Maybe I missunderstood something

File: main.nf (hunk 5 above)
Comment (Hammarn) [reply main.nf:70]: hmm, I thought I changed that. Maybe I forgot to push my latest changes?

File: main.nf (hunk 6 above)
Comment (Hammarn) [reply main.nf:109]: well, only `SENSITIVITY` as far as I can see

File: conf/uppmax.config (hunk 1 above)
Comment (ewels) [reply conf/uppmax.config:35]: meme

File: main.nf (hunk 3 above)
Comment (ewels) [reply main.nf:67]: Their variable `step` comes from `params.step` [here](https://github.com/SciLifeLab/CAW/blob/master/main.nf#L76)

File: main.nf (hunk 6 above)
Comment (ewels) [reply main.nf:109]: And `SAMPLE_NAME`?

File: conf/uppmax.config (hunk 1 above)
Comment (ewels) [reply conf/uppmax.config:35]: You reference `$fusioncatcher_data_dir` directly, but that variable name doesn't exist. If you want to get it from this config then it needs to be `$params.references.fusioncatcher_data_dir`

Comment (Hammarn) [reply conf/uppmax.config:35]: Gotcha!

**Pull request merged**